mappings/net/minecraft/world/gen/feature/StructureFeatures.mapping

```diff
@@ -0,0 +1,38 @@
+CLASS net/minecraft/unmapped/C_eheutewl net/minecraft/world/gen/feature/StructureFeatures
```

Comment:
I think we should keep it as `StructureFeatures`, since `Structure` is now another class, and the class has fields for feature holders
```java
    public static final Holder<StructureFeature> PILLAGER_OUTPOST = ...
    public static final Holder<StructureFeature> MINESHAFT = ...
    public static final Holder<StructureFeature> MINESHAFT_MESA = ...
    public static final Holder<StructureFeature> MANSION = ...
    public static final Holder<StructureFeature> JUNGLE_PYRAMID = ...
...
```